source/scripts/disruption/README.md

```diff
@@ -0,0 +1,234 @@
+
+```
+PYTHONPATH="navitia_source_dir/source/chaos-proto" python2.7 disruptor.py -b pyamqp://guest:guest@localhost:5672 -e "navitia" -t "shortterm.transilien" --empty_disruption
+```
+
+Simple disruption with pt object impacted
+
+```
+# Line - impact No SERVICE
+PYTHONPATH="navitia_source_dir/source/chaos-proto" python2.7 disruptor.py -b pyamqp://guest:guest@localhost:5672 -e "navitia" -t "shortterm.transilien" -p "Line('line_uri')"
+
+# Line section - impact NO SERVICE
+PYTHONPATH="navitia_source_dir/source/chaos-proto" python2.7 disruptor.py -b pyamqp://guest:guest@localhost:5672 -e "navitia" -t "shortterm.transilien" -p "LineSection('line_uri', 'stop_area_uri', 'stop_area_uri')"
+
+# Route - impact REDUCED SERVICE
+PYTHONPATH="navitia_source_dir/source/chaos-proto" python2.7 disruptor.py -b pyamqp://guest:guest@localhost:5672 -e "navitia" -t "shortterm.transilien" -p "Route('route_uri')" -i "REDUCED_SERVICE"
+```
+
+Disruptions file
+
+Example of file (`impact.json <https://github.com/canaltp/navitia/blob/dev/source/scripts/disruption/impacts.json>`).
+
+```
+{
+    "impacts": [
+        {
+            "pt_object": "LineSection('line:DUA:810801041', 'stop_area:DUA:SA:8775810', 'stop_area:DUA:SA:8738221')",
+            "impact_type": "NO_SERVICE"
+        },
+        {
+            "pt_object": "LineSection('line:DUA:810987658', 'stop_area:DUA:SA:8775657', 'stop_area:DUA:SA:7890861')",
+            "impact_type": "NO_SERVICE"
+        },
+        {
+            "pt_object": "LineSection('line:DUA:810806753', 'stop_area:DUA:SA:8876553', 'stop_area:DUA:SA:6252567')",
+            "impact_type": "NO_SERVICE"
+        },
+        {
+            "pt_object": "Line('line:DUA:800853022')",
+            "impact_type": "NO_SERVICE"
+        },
+        {
+            "pt_object": "Line('line:DUA:800999022')",
+            "impact_type": "NO_SERVICE"
+        },
+        {
+            "pt_object": "Route('route:DUA:80054622')",
+            "impact_type": "REDUCED_SERVICE"
+        },
+        {
+            "pt_object": "Route('route:DUA:800736022')",
+            "impact_type": "NO_SERVICE"
+        },
+        {
+            "pt_object": "StopPoint('stop_point:DUA:SP:93:1317')",
+            "impact_type": "NO_SERVICE"
+        },
+        {
+            "pt_object": "StopArea('stop_area:DUA:SA:8876553')",
+            "impact_type": "NO_SERVICE"
+        },
+        {
+            "pt_object": "Network('network:DUA855')",
+            "impact_type": "NO_SERVICE"
+        }
+    ]
+}
+```
+
+```
+# play file with sleep = 5 sec
+PYTHONPATH="navitia_source_dir/source/chaos-proto" python2.7 disruptor.py -b pyamqp://guest:guest@localhost:5672 -e "navitia" -t "shortterm.transilien" -f path_to_file -s 5
+
+# play file with sleep = 5 sec and loop forever
+PYTHONPATH="navitia_source_dir/source/chaos-proto" python2.7 disruptor.py -b pyamqp://guest:guest@localhost:5672 -e "navitia" -t "shortterm.transilien" -f path_to_file -s 5 -l
+```
```

Comment:
You my add here:
"# All the object codes in the above command line examples as well as in file 'impacts.josn' will have to be replaced by the object codes existing in your kraken (instance). For a LineSection, StopAreas should be related to the Line.

Reply:
yes my bad, 
we must have the jormungandr virtualenv to run the script